e2e-cypress/files/README.md

```diff
@@ -23,6 +40,11 @@ In order to generate one xml report per test type (installation, integration and
 
 ## E2e test user authentication
 
+With Cypress 12 version is now available `cy.origin()` that allows you to handle redirections. This funcionality eases the login handling.
```

Comment:
This is SOOO cool! Didn't know this is possible now. Gave me some crazy rides with some puppeteer embedded into Cypress in the past 🤪